### ISSUE TITLE
Problem with href links

### ISSUE THREAD

**Issue by devsoftwaresrl** [adgsm/multi-level-push-menu#159]
Hi,
I tried to implemented this component in my website but I don't understand why the HREF LINK doesn't work correctly.
If one voice is composed by one level menu with external link (example: www.google.it), why the link doesn't work?

Thanks.

**Comment by adgsm** [comment]
Take a look at #102

**Comment by devsoftwaresrl** [comment]
I don't understand ... where can I edit the menu's javascript parameters?
In Which file?
Where and what do I insert into the code to make clickable links and leading to a page?
In the demo I downloaded there isn't anything.

Thank you

**Comment by adgsm** [comment]
If you do not need callbacks initialize the menu with the following.

```
preventItemClick: false // set to false if you do not need event callback functionality
```

**Comment by devsoftwaresrl** [comment]
I need both of them, the menu voice of "back" and the other menu voices clickable.

**Comment by adgsm** [comment]
Well then keep preventItemClick set to true and use code example provided both in documentation [(here)](http://multi-level-push-menu.make.rs/#events) and in #102.
It is an copy/paste example provided in both.

```
$( '#menu' ).multilevelpushmenu({
    onItemClick: function() {
        // First argument is original event object
        var event = arguments[0],
            // Second argument is menu level object containing clicked item (<div> element)
            $menuLevelHolder = arguments[1],
            // Third argument is clicked item (<li> element)
            $item = arguments[2],
            // Fourth argument is instance settings/options object
            options = arguments[3];

        // You can do some cool stuff here before
        // redirecting to href location
        // like logging the event or even
        // adding some parameters to href, etc...

        // Anchor href
        var itemHref = $item.find( 'a:first' ).attr( 'href' );
        // Redirecting the page
        location.href = itemHref;
    }
});
```

**Comment by devsoftwaresrl** [comment]
Ok. Where should I put this code? Before the my html code, inside the tag <script></ scripit> ?

**Comment by adgsm** [comment]
It is all described in documentation at the following address [http://multi-level-push-menu.make.rs/](http://multi-level-push-menu.make.rs/).

Once you come to the initialization point [http://multi-level-push-menu.make.rs/#initialize](http://multi-level-push-menu.make.rs/#initialize) paste the code provided above instead of ```$('#menu').multilevelpushmenu();```.

**Comment by devsoftwaresrl** [comment]
Submitted code but href link still doesn't work: why???
.....    <body>

<script>
    $(document).ready(function(){
        $( '#menu' ).multilevelpushmenu({
    onItemClick: function() {
        // First argument is original event object
        var event = arguments[0],
            // Second argument is menu level object containing clicked item (<div> element)
            $menuLevelHolder = arguments[1],
            // Third argument is clicked item (<li> element)
            $item = arguments[2],
            // Fourth argument is instance settings/options object
            options = arguments[3];

        // You can do some cool stuff here before
        // redirecting to href location
        // like logging the event or even
        // adding some parameters to href, etc...

        // Anchor href
        var itemHref = $item.find( 'a:first' ).attr( 'href' );
        // Redirecting the page
        location.href = itemHref;
    }
});
    });
</script>

<div id="menu">
	<nav>
				<h2><i class="fa fa-reorder"></i>Menu</h2>
		
						<ul>
							<li><a href="http://www.sit-web.it/">test 1</a></li>							    
						</ul>
				
	</nav>
</div>



</body>


.....

**Comment by adgsm** [comment]
I have no idea really. Try looking for errors in your console...

**Comment by devsoftwaresrl** [comment]
http://www.sit-web.it/images/test.png

**Comment by devsoftwaresrl** [comment]
Href links work only if you click with the right mouse button and make "_blank" ....
Why?

Thank you

**Comment by adgsm** [comment]
Here is a working fiddle that maybe can help you (I have used window.open in a script instead of location.href which will not work in there. For your local implementation change it to location.href).

Click "Collections" or "Credits" items at the bottom of the menu. First will open google.com, and second one google.it
[http://jsfiddle.net/trfeLc4z/1/](http://jsfiddle.net/trfeLc4z/1/)

I can not help you with your local implementation.

**Comment by devsoftwaresrl** [comment]
I looked this demo.
I tried to replicate it but if I insert the javascript code as the example gives error.
In the picture "MENU 1" the menu voices are visible but href links don't go and the js code aren't complete.
http://www.sit-web.it/images/menu-1.png

In the picture "MENU 2" js code is the same of the demo but the menu voices disappear.
http://www.sit-web.it/images/menu-2.png

It Seems That it gives error when i insert this at the beginning:
" $ (function () { $ (Document) .ready (function () {...........}); }); "

Why?? 
thanks
